### DISCUSSION TITLE
Unable to run powershell scripts

### DISCUSSION THREAD

**Discussion by faizanbashir** [actions/actions-runner-controller#3483] [Questions]
Hi,

In the actions spec, we are calling build scripts which are triggered by a PowerShell. I am getting the following error while I try to execute the PowerShell scripts, it says Powershell is not installed. 

```shell
Run pwsh bootstrap.ps1
  pwsh bootstrap.ps1
  shell: /usr/bin/bash -e {0}
/home/runner/_work/_temp/[2](https://github.com/<my-organisation>/<repo-name>/actions/runs/8890924951/job/24411843335#step:4:2)7249ab3-2e3d-4830-af51-db[3](https://github.com/<my-organisation>/<repo-name>/actions/runs/8890924951/job/24411843335#step:4:3)6ffd969c1.sh: line 1: pwsh: command not found
```

How can i fix this issue? Any help is appreciated.

**Comment by jca27** [comment]
Not sure if you still need it, but you just need to install the version of powershell that gha uses.
If its your runner, just bake the following into the image. If not, add the following in a step and refresh your env.
`iex "& { $(irm https://aka.ms/install-powershell.ps1) } -UseMSI -Quiet"`
Link for more options: https://www.thomasmaurer.ch/2019/07/how-to-install-and-update-powershell-7/
